python/HeaderService/hutils.py

```diff
@@ -97,15 +97,14 @@ def read_head_template(fname, header=None):
     with open(fname) as fobj:
         lines = fobj.readlines()
 
-    # Ignore empty keyword
-    lines = [l.rstrip() for l in lines if l[0:3] != 'END' and l[0:8] != ' '*8]
+    lines = [l.rstrip() for l in lines if l[0:3] != 'END']
 
     # if header is None an empty FITSHDR is created
     hdr = fitsio.FITSHDR(header)
 
     for l in lines:
         hdr.add_record(l)
-
+        
```

Comment:
This extra space should not be included. Are you running flake8?

Reply:
Fixed.